.github/prettierignore

```diff
@@ -0,0 +1,29 @@
+ISSUE_TEMPLATE
```

Comment:
In support of making it easier for contributors to run Prettier themselves, shouldn't this file be at the top-level of the repo, named `.prettierignore`? (Cf. https://prettier.io/docs/en/ignore.html)